build/resources/gpii-linux.service

```diff
@@ -0,0 +1,14 @@
+# Use with Systemd > 219
+[Unit]
+Description=omponents of the GPII personalization infrastructure for use on Linux
+Documentation=http://gpii.net
+
+[Service]
+Type=simple
+ExecStart=/usr/bin/gpii-linux-start
```

Comment:
This Systemd unit uses the [Systemd User](https://wiki.archlinux.org/index.php/Systemd/User) feature to launch GPII with user pemissions at log in time. The environment variables like dbus socket or policykit resources are set by pam.d at login time and retrieved by the command `systemctl --user import-environment` when the service is launched. The ansible-nodejs wasn't made for the previous purpose. For example, the `user` and `group` statements don't make sense here because, Systemd will set the user's UID and GID to the child processes.

Anyway this unit can only be use in >=Fedora 23 because Fedora 22 has a minor version of Systemd that doesn't have that feature.